test/provision/manifest/1.14/coredns_deployment.yaml

```diff
@@ -71,13 +73,19 @@ data:
             fallthrough in-addr.arpa ip6.arpa
         }
         proxy cilium.test 10.96.0.100:53 {
-          fail_timeout 10s
-          max_fails 0
+            fail_timeout 10s
+            max_fails 0
         }
         proxy . /etc/resolv.conf {
-          fail_timeout 10s
-          max_fails 0
+            fail_timeout 10s
+            max_fails 0
         }
+        prometheus :9153
+        forward . /etc/resolv.conf
+        cache 30
```

Comment:
This cache will make it hard to have in the CI, in these .3 version it behaves different than .2

Reply:
What does this cache parameter do?

Reply:
It keeps the value in the coredns for 30 seconds, but in the version .2 the data was updated if kubernetes watcher send something new, I do not think that in .3 version do that, so the dns can resolve to an invalid name and test will fail. (Happens to me in 1.14 testing)